src/transformers/utils/bitsandbytes.py

```diff
@@ -0,0 +1,98 @@
+from transformers.utils import is_accelerate_available, is_bitsandbytes_available
+
+
+if is_bitsandbytes_available():
+    import torch
+    import torch.nn as nn
+
+    import bitsandbytes as bnb
+
+if is_accelerate_available():
+    from accelerate import init_empty_weights
+
+
+def set_module_8bit_tensor_to_device(module, tensor_name, device, value=None):
+    """
+    A helper function to set a given tensor (parameter of buffer) of a module on a specific device (note that doing
+    `param.to(device)` creates a new tensor not linked to the parameter, which is why we need this function). The
+    function is adapted from `set_module_tensor_to_device` function from accelerate that is adapted to support the
+    class `Int8Params` from `bitsandbytes`.
+
+    Args:
+        module (`torch.nn.Module`):
+            The module in which the tensor we want to move lives.
+        tensor_name (`str`): The full
+            name of the parameter/buffer.
```

Comment:
Weird formatting.

Reply:
Thanks for spotting this ;) ! Fixed in 9bd326b7eaf758e19f12166ec0bbba1906d53062